packages/block-editor/src/hooks/block-bindings.js

```diff
@@ -0,0 +1,69 @@
+/**
+ * WordPress dependencies
+ */
+import { __ } from '@wordpress/i18n';
+import { store as blocksStore } from '@wordpress/blocks';
+import {
+	PanelBody,
+	__experimentalItemGroup as ItemGroup,
+	__experimentalItem as Item,
+} from '@wordpress/components';
+import { useSelect } from '@wordpress/data';
+
+/**
+ * Internal dependencies
+ */
+import { store as blockEditorStore } from '../store';
+import { unlock } from '../lock-unlock';
+import InspectorControls from '../components/inspector-controls';
+
+export const BlockBindingsPanel = () => {
+	const { bindings } = useSelect( ( select ) => {
+		const { getSelectedBlockClientId, getBlock } =
+			select( blockEditorStore );
+		const _selectedBlockClientId = getSelectedBlockClientId();
```

Comment:
If I am not mistaken, the block props are passed to the component, so we should be able to access the clientId without using this function. Something like:

```
export const BlockBindingsPanel = ( { clientId } )
```

I am not 100% sure but maybe it is worth trying.

Reply:
I would also check the context as it potentially has the bindings exposed, too.

Reply:
https://github.com/WordPress/gutenberg/blob/44bca551553b808526ff112b295328457a076b44/packages/block-editor/src/components/block-edit/index.js#L69

Yes, it's also more performant to access now bindings through context.